docs/Text-Utils.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Typography</title>
```

Comment:
This and your typography have same title. please, see to it.

Reply:
Resolved in the latest commit. Thank You.